config/routes.rb

```diff
@@ -9,6 +9,9 @@
     resources :home, only: [:index]
     resources :adoptable_pets, only: %i[index show]
     resources :faq, only: [:index]
+    resources :activations do
+      patch "update_activation"
```

Comment:
we don't need this custom route anymore (that was the intention of moving this to its own resource). So we can can keep it RESTful and use the `update` action on this controller.